modules/virtualization/microvm/guivm.nix

```diff
@@ -118,6 +118,14 @@
   };
   cfg = config.ghaf.virtualization.microvm.guivm;
   vsockproxy = pkgs.callPackage ../../../packages/vsockproxy {};
+
+  # Importing kernel builder function from packages and checking hardening options
+  buildKernel = import ../../../packages/kernel {inherit config pkgs lib;};
+  config_baseline = ../../hardware/x86_64-generic/kernel/configs/ghaf_host_hardened_baseline-x86;
+  # TODO: Find proper way to place configs, below are commented as its causing build errors
```

Comment:
This should be set in the target configuration - similar to the module tests. I think you can remove these from here.